pkg/volume/plugins.go

```diff
@@ -129,6 +129,13 @@ type AttachableVolumePlugin interface {
 	NewDetacher(name string, podUID types.UID) (Detacher, error)
 }
 
+// VolumeLabelerPlugin is an extended interface of VolumePlugin and is used to label persistentvolumes
+// with metadata from external cloud.
```

Comment:
Updated to `is used to get default labels for PersistentVolumes from external cloud.`